src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/MessageToStringConverter.cs

```diff
@@ -20,6 +20,10 @@ internal class MessageToStringConverter : IAsyncConverter<Message, string>
             {
                 throw new ArgumentNullException("input");
             }
+            if (input.Body == null)
```

Comment:
What happens end to end when null is returned here? Does the user function get invoked with a null or empty string?